webrtc-charter.html

```diff
@@ -294,24 +305,19 @@ <h3>
               ended on 19 July 2015<br>Produced under <a href="https://www.w3.org/2011/04/webrtc-charter.html">2011 WebRTC Working Group Charter</a></dd>
-            <dt class="spec"><a href="https://w3c.github.io/webrtc-dscp-exp/">DSCP Control API</a></dt>
-            <dd>An extension to the WebRTC 1.0 API to enable experimentation with QoS marking.</dd>
-            <dd>Draft state: <a href="https://w3c.github.io/webrtc-dscp-exp/">Editor's draft</a></dd>
           </dl>
 
-<p>The preferred mode of adding new features is by means of extension specifications, rather than new features in the existing specs. At appropriate times, the WG may choose to redistribute the work between chartered documents - by splitting, merging or repartitioning the work as appropriate.</p>
-
-<p>By the end of the charter period, the WG expects to have a small number of "core" specifications, which allows useful applications to be built using only those specifications, and a series of "extension" specifications that extend the applicability of the APIs specified to more applications.</p>
-
-<p>This work will be done in collaboration with the IETF. The W3C will define APIs to ensure that application developers can control the components or the architecture for selection and profiling of the wire protocols that have been produced by the IETF Real-Time Communication in WEB-browsers (RTCWeb) Working Group. While the specified API Functions will not constrain implementations into supporting a specific profile, they will be compatible with the Profile that is specified by the RTCWeb Working Group.</p>
-
-<p>The highest priority of the group in this charter period will be to get the core recommendations progressed towards Recommendation status.</p>
+          <p>The Working Group will maintain the following W3C Recommendations (gathering of errata, publication of new editions incorporating errata):</p>
+          <ul>
+            <li><a href="https://www.w3.org/TR/webrtc/">WebRTC 1.0: Real-time Communication Between Browsers</a></li>
+            <li><a href="https://www.w3.org/TR/mediacapture-streams/">Media Capture and Streams</a></li>
+          </ul>
 
-<p>As the name indicates, WebRTC 1.0: Real-time Communication Between Browsers is a first version of APIs for real-time communication, sometimes referred to as the PeerConnection API. The activities in the ORTC (Object Real-time Communications) Community Group indicate that there is interest in additional APIs to provide more direct control over WebRTC than what the PeerConnection API offers.</p>
+<p>The preferred mode of adding new features is by means of extension specifications, rather than new features in the existing specs. At appropriate times, the WG may choose to redistribute the work between chartered documents - by splitting, merging or repartitioning the work as appropriate.</p>
```

Comment:
Is this subject to change if W3C migrates to more of a live document model? Does this leave us an out?

Reply:
If they adopt a living model, we can interpret "as appropriate" as "continuously".